alws/crud/build.py

```diff
@@ -198,7 +198,8 @@ async def generate_query(count=False):
         return query
 
     if build_id:
-        return await db.execute(await generate_query()).scalars().first()
```

Comment:
I believe that `return (await db.execute(await generate_query())).scalars().first()` should do the trick too

Reply:
But hey, not a strong opinion on doing that way or the other ;)

Reply:
I don't know why, but albs started working after this change. I've checked twice.